rules/stylistic-issues.js

```diff
@@ -30,64 +30,64 @@ module.exports = {
+		"new-parens": "error", // disallow the omission of parentheses when invoking a constructor with no arguments
+		"newline-after-var": [ "off", "always" ], // require or disallow an empty newline after variable declarations
+		"newline-before-return": "off", // require an empty line before return statements
+		"newline-per-chained-call": "off", // require a newline after each call in a method chain
+		"no-array-constructor": "error", // disallow use of the Array constructor
+		"no-bitwise": "off", // JSHINT disallow use of bitwise operators
+		"no-continue": "error", // disallow use of the continue statement
+		"no-inline-comments": "off", // disallow comments inline after code
+		"no-lonely-if": "error", // disallow if as the only statement in an else block
+		"no-mixed-operators": "error", // disallow mixed binary operators
+		"no-mixed-spaces-and-tabs": "error", // JSCS disallow mixed spaces and tabs for indentation
+		"no-multiple-empty-lines": [ "error", { max: 2 } ], // JSCS disallow multiple empty lines
+		"no-negated-condition": "off", // TRIAL disallow negated conditions
+		"no-nested-ternary": "error", // disallow nested ternary expressions
+		"no-new-object": "error", // disallow the use of the Object constructor
+		"no-plusplus": "off", // JSHINT disallow use of unary operators, ++ and --
+		"no-restricted-syntax": [ "error", "WithStatement" ], // disallow use of certain syntax in code
+		"no-tabs": "off", // disallow tabs in file
+		"no-ternary": "off", // disallow the use of ternary operators
+		"no-trailing-spaces": "error", // JSCS disallow trailing whitespace at the end of lines (fixable)
+		"no-underscore-dangle": "off", // disallow dangling underscores in identifiers
+		"no-unneeded-ternary": "error", // disallow the use of ternary operators when a simpler alternative exists
+		"no-whitespace-before-property": "error", // disallow whitespace before properties
+		"object-curly-newline": "off", // enforce consistent line breaks inside braces
+		"object-curly-spacing": [ "error", "always" ], // require or disallow padding inside curly braces (fixable)
+		"object-property-newline": "off", // enforce placing object properties on separate lines
+		"one-var-declaration-per-line": "off", // require or disallow newlines around var declarations
+		"one-var": [ "error", { "initialized": "never", "uninitialized": "always" } ], // JSCS require or disallow one variable declaration per function
+		"operator-assignment": [ "error", "always" ], // require assignment operator shorthand where possible or prohibit it entirely
+		"operator-linebreak": [ "error", "after", { "overrides": { "?": "after", ":": "after" } } ], // JSCS enforce operators to be placed before or after line breaks
+		"padded-blocks": [ "error", "never" ], // JSCS enforce padding within blocks
+		"quote-props": [ "error", "as-needed", { "keywords": true } ], // JSCS require quotes around object literal property names
```

Comment:
Please put `keywords` back to `false`. Apparently JSCS was not fixing this before because now ESLint is fixing it. At this point we probably shouldn't introduce changes intentionally, but only keep fixing things we used to fix.